### PR TITLE
Bec63 beacon starter

### PR DESCRIPTION
**Pull request by acenolaza**
This is a first attempt to get something running in order to read a beacon value from an already deployed set of contracts.

The main idea was to identify what were the requirements for the scripts in this repo that will need to be then added to the services repo. For instance so far I know that we need to find a way to whitelist the BeaconReaderExample contract on the deployed RrpBeaconServer in order to read it's values. For now I've added a whitelister contract that needs to be moved to the same repo where the RrpBeaconServer is being deployed (beacon-setup-guide?)

I'm also using a localhost.json file for testing the scripts but this needs to be deleted and replaced with an import from the services repo.

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  }
+});
+
+// You need to export an object to set up your config
+// Go to https://hardhat.org/config/ to learn more
+const config: HardhatUserConfig = {
+  solidity: "0.8.9",
+  networks: {
+    ropsten: {
+      url: "https://eth-ropsten.gateway.pokt.network/v1/lb/61d5f1ff1a94ee003a8e070c",
```
Comment (aquarat) [inline hardhat.config.ts:20]: Hi, this looks like a valid key? It probably shouldn't be here?

File: hardhat.config.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      accounts: {
+        mnemonic:
+          "tube spin artefact salad slab lumber foot bitter wash reward vote cook",
```
Comment (aquarat) [inline hardhat.config.ts]: Is this also a problem?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,46 @@
...
+
+This project is composed of the following steps:
+
+1. Create a template that will save a beconId to a file under the `/beacons` directory
+1. **TODO:NEEDS TO BE MOVED TO beacon-setup-guide repo** Deploy RrpBeaconServerReaderWhitelister
+1. Deploy BeaconReaderExample contract and fetch the beacon value from the RrpBeaconServer contract by beaconId
+
+**WARNING:** This project is supposed to use RRP contracts already deployed on a supported chain with a [RrpBeaconServerReaderWhitelister](beacon-starter/contracts/RrpBeaconServerReaderWhitelister.sol) contract deployed. This contract will be called by the [`read-beacon-value.ts`](beacon-starter/scripts/read-beacon-value.ts) script in order to whitelist the reader contract on the [RrpBeaconServer](https://github.com/api3dao/airnode/blob/master/packages/airnode-protocol/contracts/rrp/requesters/RrpBeaconServer.sol#L117) contract. Currenlty this is consuming a [test deployment file](beacon-starter/deployments/0.3.1/localhost.json) that will try to interact with locally deployed contracts. Once the services repo is ready this file should be removed and scripts must be updated to connect to those contracts instead.
```
Comment (aquarat) [inline README.md:11]: This link appears to be broken? (beacon-starter*)

File: hardhat.config.ts (hunk 1 above)
Comment (acenolaza) [reply hardhat.config.ts:20]: It's just a test api key I've created for this but you've made realized that this needs to be in .env file anyways 🥇

File: hardhat.config.ts (hunk 2 above)
Comment (acenolaza) [reply hardhat.config.ts]: Same as before it should go into .env file but this is our test account and this mnemonic is in all the config.json.example files as well as airnode-examples

Comment (aquarat) [reply hardhat.config.ts]: Ah thanks, it did look vaguely familiar.

File: test/BeaconReaderExample.test.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    // Deploy BeaconReaderExample contract
+    const BeaconReaderExample = await ethers.getContractFactory(
+      "BeaconReaderExample"
+    );
+    const beaconReaderExample = await BeaconReaderExample.deploy(
+      "0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e"
+    );
+    await beaconReaderExample.deployed();
+
+    // TODO: create a template like in operation pacakge (add script too)
```
Comment (aquarat) [inline test/BeaconReaderExample.test.ts]: Nit, spelling.

**Review by aquarat — APPROVED** [review]
It's difficult to "properly" review this because it relies on placeholders, so it's difficult to work through the guide locally, but generally it looks fine to me with these limitations taken into account.

There are a lot of hardcoded values which I assume will be swapped out for external resources/ENVs. :)

**Pull request closed without merging**